aesm-client/src/lib.rs

```diff
@@ -218,6 +265,27 @@ impl AesmClient {
             attributes,
         )
     }
+
+    pub fn init_quote_ecdsa_p256(&self) -> Result<QuoteInfoEx> {
```

Comment:
Remove this wrapper, you should expose `get_supported_att_key_ids` and `init_quote_ex` directly

Reply:
removed, thanks